web-interface/vm-consoles-modules/vm-desktop-viewer-web.adoc

```diff
@@ -0,0 +1,15 @@
+[[vm-desktop-viewer-web]]
+== Using Desktop Viewer to connect to RDP Console (Windows only)
+
+Connect with *Desktop Viewer* to running virtual machine from the *Consoles* tab in the *Virtual Machine Details* screen of the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine.
+. Click *Consoles*. The VNC console opens by default.
+. Select *Desktop Viewer* in drop-down list.
+. Choose nework interface you want to connect to from *Network Interface* drop-down.
+.. If L2 Network Interface is chosen, you need to install qemu-guest-agent on VM
+.. If Pod network interface is chosen, you need to create a Service to expose VM port 3389 (RDP) - follow onscreen instructions to do so.
```

Comment:
Small changes -> "If the pod network interface is chosen, you will need to create a service to expose port 3389 (RDP) on the VM. Follow the on-screen instructions to do so."